doc/related_projects.rst

```diff
@@ -182,6 +182,8 @@ and tasks.
   (`CRFsuite <http://www.chokkan.org/software/crfsuite/>`_ wrapper with
   sklearn-like API).
 
+- `skforecast <https://github.com/JoaquinAmatRodrigo/skforecast>`_ A python library that eases using scikit-learn regressors as multi-step forecasters. It also works with any regressor compatible with the scikit-learn API.
```

Comment:
@seoeunHong May you wrap this line so that it uses < 88 characters per line?

@JoaquinAmatRodrigo What do you think of this description for skforecast?

Reply:
@thomasjpfan Thank you so much for reviewing my code! I wrapped my code lines.